README.rst

```diff
@@ -4,8 +4,55 @@ Platform Plugin Aspects
 Purpose
 *******
 
-This repository holds various Aspects related plugins for edx-platform.
+This repository holds various Aspects plugins for the Open edX platform.
 
+Sinks
+*****
+
+Sinks are components that listen for events and store them in ClickHouse. The
+events are emitted by the Open edX platform via `Open edX events`_ or Django signals.
+
+Available Sinks
+===============
+
+- `CourseOverviewSink` - Listens for the `COURSE_PUBLISHED` event and stores the
+  course structure data through `XBlockSink` in ClickHouse.
+- `ExternalIdSink` - Listens for the `post_save` Django signal on the `ExternalId`
+  model and stores the external id data in ClickHouse.
+- `UserProfile` - Listens for the `post_save` Django signal on the `UserProfile`
+  model and stores the user profile data in ClickHouse.
+- `UserRetirementSink` - Listen for the `USER_RETIRE_LMS_MISC` Django signal and
+  remove the user PII information from ClickHouse.
+
+Commands
+========
+
+In addition to being an event listener, this package provides the following commands:
+
+- `dump_data_to_clickhouse` - This command allows bulk export of the data from the Sinks.
+  Allows bootstrapping a new data platform or backfilling lost or missing data.
+
+    ``python manage.py cms dump_courses_to_clickhouse``
+
+- `load_test_tracking_events` - This command allows loading test tracking events into
```

Comment:
Can you link these next two to the how-to doc `load_test_tracking_pipeline.rst`?